paddle/fluid/framework/ir/mkldnn/conv_int8_scales_pass.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+// #include <memory>
+// #include <string>
+// #include <unordered_map>
+
+#include "paddle/fluid/framework/ir/fuse_pass_base.h"
+
+namespace paddle {
+namespace framework {
+namespace ir {
+
+class Graph;
+/*
+ * compute quantization scales for biases and weights
+ */
+class ConvInt8ScalesPass : public FusePassBase {
```

Comment:
I'm wondering if we should change the name for that pass, first, this pass is connected only with mkldnn ops, so maybe it should consist of `mkldnn` word, second this pass later can be extended with scale calculation for FC op. 
My proposition: int8_scale_mkldnn_pass or int8_scale_calculation_pass. Please share your opinion.

Reply:
another possibility is `mkldnn_int8_scale_pass` but `int8_scale_mkldnn_pass` seems slightly better for me

Reply:
We mostly have mkldnn at the end of names of passes so I like `scale_calculation_mkldnn_pass` or `int8_scale_calculation_mkldnn_pass`

Reply:
I would go for `int8_scale_calculation_mkldnn_pass` as there is also scale op and the first proposition could be misleading